src/databricks/labs/ucx/install.py

```diff
@@ -598,7 +598,8 @@ def _create_dashboard(self, folder: Path, *, parent_path: str) -> None:
         """Create a lakeview dashboard from the SQL queries in the folder"""
         logger.info(f"Creating dashboard in {folder}...")
         metadata = DashboardMetadata.from_path(folder).replace_database(
-            database=self._config.inventory_database, database_to_replace="inventory"
+            database=f"hive_metastore.{self._config.inventory_database}",
```

Comment:
This is how `WorkspaceConfig.replace_inventory_variable` replaced the database

Reply:
I believe we want to use a configured metastore rather than hardocoded "hive_metastore" ?

Reply:
At the moment UCX is hardcoded pretty much everywhere to use the `hive_metastore` catalog; I don't believe it's configurable.

Reply:
it is not configurable atm, work for another PR